resources/documentation/multiple_icebergs.rst

```diff
@@ -35,7 +35,7 @@ Few notes about this schema:
 
 * Icebergs can be connected to any administration networks.
 * Managements of a standard iceberg are expected to be connected to both administration network,
-i.e. on of their internal iceberg administration network, but also on one of the administration network of their master iceberg.
+  i.e. on of their internal iceberg administration network, but also on one of the administration network of their master iceberg.
```

Comment:
typo? i.e. on **one** of their internal...

Reply:
Good catch, solved.